From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 23 Jan 2017 14:34:01 +0000
Subject: power: Fix ABI change in 4.9.3
Forwarded: not-needed

Commit bed570307ed7 "PM / wakeirq: Fix dedicated wakeirq for drivers
not using autosuspend" changed struct wake_irq, but it is only used
by built-in code and its definition is not exposed to OOT modules
at all.  Hide the change from genksyms.

---

--- a/drivers/base/power/power.h
+++ b/drivers/base/power/power.h
@@ -28,8 +28,13 @@ extern void pm_runtime_remove(struct dev
 
 struct wake_irq {
 	struct device *dev;
+#ifndef __GENKSYMS__
 	unsigned int status;
+#endif
 	int irq;
+#ifdef __GENKSYMS__
+	bool dedicated_irq:1;
+#endif
 };
 
 extern void dev_pm_arm_wake_irq(struct wake_irq *wirq);
